src/main/java/se/citerus/dddsample/interfaces/tracking/ws/CargoTrackingDTOConverter.java

```diff
@@ -16,7 +16,7 @@
 
 public class CargoTrackingDTOConverter {
     private static final DateTimeFormatter formatter = DateTimeFormatter
-            .ofLocalizedDateTime(FormatStyle.MEDIUM)
+            .ofPattern("MMM d, uuuu, h:mm:ss a", Locale.ENGLISH) // See https://github.com/spring-projects/spring-framework/wiki/Date-and-Time-Formatting-with-JDK-20-and-higher#recommendations
```

Comment:
This is a good catch (FormatStyle.MEDIUM does seem like a bad choice given it's locale-dependent behavior) but how about using an ISO-8601 date pattern like `YYYY-MM-DDTHH:MM:SSZ` instead?

Reply:
imho, the ISO format is way better but i thought you had a specific reason for choosing this format.

Reply:
I used the ISO format for values but kept this format for descriptions since it's more readable